src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] As it is in the project requirements, kindly note that the action types are supposed t be implemented using the [duck's pattern](https://github.com/erikras/ducks-modular-redux). Kindly implement this.